docs/club/competition-tool/plugin-settings.md

```diff
@@ -95,34 +94,33 @@ Automatically enabled for all matches.
 
-| Setting name                 | Type    | Default value   | Description                        |
-|:-----------------------------|:--------|:----------------|:-----------------------------------|
-| S_UsePlayerMinimumRankPlugin | Boolean | `False`         | Enable the plugin.                 |
-| S_MinimumRank                | String  | `0`             | The rank to reach to be qualified. |
-
+| Setting name                 | Type    | Default value | Description                        |
+| :--------------------------- | :------ | :------------ | :--------------------------------- |
+| S_UsePlayerMinimumRankPlugin | Boolean | `False`       | Enable the plugin.                 |
+| S_MinimumRank                | String  | `0`           | The rank to reach to be qualified. |
 
 ## Time limit
 
 Limit the duration a player can stay on a server.
 
-| Setting name         | Type    | Default value  | Description                                                                                                    |
-|:---------------------|:--------|:---------------|:---------------------------------------------------------------------------------------------------------------|
-| S_UseTimeLimitPlugin | Boolean | `False`        | Enable the plugin.                                                                                             |
-| S_TimeLimitDuration  | Integer | `300`          | The duration a player can stay on the server in seconds, based on the player's first connection to the server. |
+| Setting name         | Type    | Default value | Description                                                                                                    |
+| :------------------- | :------ | :------------ | :------------------------------------------------------------------------------------------------------------- |
+| S_UseTimeLimitPlugin | Boolean | `False`       | Enable the plugin.                                                                                             |
+| S_TimeLimitDuration  | Integer | `300`         | The duration a player can stay on the server in seconds, based on the player's first connection to the server. |
 
 ## Trackmania UI
 
-| Setting name                | Type   | Default value   | Description                                            |
-|:----------------------------|:-------|:----------------|:-------------------------------------------------------|
-| S_TrackmaniaUIProperties    | String | `""`            | The UI properties to hide. Add value to enable the plugin. |
+| Setting name             | Type   | Default value | Description                                                |
+| :----------------------- | :----- | :------------ | :--------------------------------------------------------- |
+| S_TrackmaniaUIProperties | String | `""`          | The UI properties to hide. Add value to enable the plugin. |
```

Comment:
Afaik this was made very early in TM2 days and is no longer functional in TMNext, would be good to mark it as deprecated in some way. Could be an improvement from Nadeo's side to make it working again using the `Common.UIModules.SetProperties` mode script method 🤔